apps/cli/src/commands/boot-challenger.ts

```diff
@@ -114,26 +115,78 @@ const onAssertionConfirmedCb = async (nodeNum: any, commandInstance: Vorpal.Comm
+    const challengeData = await getLatestChallenge();
+    const lastChallenge = challengeData[1];
+
+    const lastAssertionId = Number(lastChallenge.assertionId);
+    const time = Number(lastChallenge.assertionTimestamp);
+
+    // Calculate the minimum time to submit an assertion
+    const minimumTimeToSubmit = time + MINIMUM_TIME_BETWEEN_ASSERTIONS;
+
+    // Check if enough time has passed that we can submit an assertion
+    if(Date.now() > minimumTimeToSubmit) {
+        let confirmDataToSubmit: string;
+
+        const assertionNode = await getAssertion(nodeNum);
+        commandInstance.log(`[${new Date().toISOString()}] Assertion data retrieved. Starting the submission process...`);
+
+        // Determine if we need to submit a batch challenge
+        const isBatchChallenge = lastAssertionId + 1 !== nodeNum;
+
+        if(isBatchChallenge) {
+
+            // If we are submitting for multiple assertions, get all of the assertion ids
+            const assertionIds = Array.from({ length: nodeNum - lastAssertionId }, (_, i) => lastAssertionId + i + 1);    
+            commandInstance.log(`[${new Date().toISOString()}] Submitting a batch challenge for assertion ids ${assertionIds}.`);
+
+            try {
+                // Get the confirm data for all of the assertions
+                const [confirmData, confirmDataHash] = await getMultipleChallengeConfirmData(assertionIds);
+
+                // Set the confirm data to submit for a batch challenge
+                confirmDataToSubmit = confirmDataHash;
+
+            } catch (error) {
+                commandInstance.log(`[${new Date().toISOString()}] Error Getting Batch Confirm Data Error: Assertion Ids => ${assertionIds} ${(error as Error).message}`);
+                sendNotification(`Get Batch Confirm Data Error: Assertion Ids => ${assertionIds} ${(error as Error).message}`, commandInstance);
+                throw error;
+            }
+        }else{
+            commandInstance.log(`[${new Date().toISOString()}] Submitting a challenge for assertion id ${nodeNum}.`);
+
+            // Set the confirm data to submit for a single challenge
+            confirmDataToSubmit = assertionNode.confirmData;
         }
-        commandInstance.log(`[${new Date().toISOString()}] Submit Assertion Error: ${(error as Error).message}`);
-        sendNotification(`Submit Assertion Error: ${(error as Error).message}`, commandInstance);
-        throw error;
+
+        try {
+            // Submit a Batched Challenge
+            await submitAssertionToReferee(
+                cachedSecretKey,
+                nodeNum,
+                lastAssertionId,
+                confirmDataToSubmit,
+                assertionNode.createdAtBlock,
+                cachedSigner!.signer,
+            );
+
+            // Update the last submitted assertion time
+            lastAssertionTime = Date.now();
+
+            commandInstance.log(`[${new Date().toISOString()}] Submitted assertion: ${nodeNum}`);
```

Comment:
@CryptITAustria previous feedback was to update this log to "tell us if multiple got submitted".

This already happens on line 142 for batch, or 157 for a single. I just wanted to confirm if you still also want it included here?